build.gradle

```diff
@@ -51,13 +51,13 @@ ext {
   ehcache2Version = '2.10.3'
 
   // Clustered
-  terracottaPlatformVersion = '5.3.0-pre2'
+  terracottaPlatformVersion = '5.3.0-pre3'
```

Comment:
will be 5.3.0-pre4 now because I'll require https://github.com/Terracotta-OSS/terracotta-platform/pull/308